huaweicloud/services/swr/resource_huaweicloud_swr_image_trigger.go

```diff
@@ -169,7 +169,7 @@ func resourceSwrImageTriggerCreate(ctx context.Context, d *schema.ResourceData,
 		},
 	}
 	clusterName := d.Get("cluster_name").(string)
-	if clusterName == "" {
+	if clusterName == "" && d.Get("type").(string) != "cci" {
```

Comment:
how about just restricting to cce here, just in case we have other types supported in future.

Reply:
fixed